api/v1/controllers/RegistrationController.js

```diff
@@ -1,17 +1,45 @@
+var _ = require('lodash');
 var bodyParser = require('body-parser');
+var _Promise = require('bluebird');
 
 var services = require('../services');
 var middleware = require('../middleware');
 var requests = require('../requests');
 var roles = require('../utils/roles');
 var mail = require('../utils/mail');
+var registration = require('../utils/registration');
+var errors = require('../errors');
 
 var router = require('express').Router();
 
 function _isAuthenticated (req) {
 	return req.auth && (req.user !== undefined);
 }
 
+function _validateGetAttendeesRequest(page, count, category, ascending){
```

Comment:
Not really. I'd be fine with just keeping this for now and adding query param validation in a separate branch.